### DISCUSSION TITLE
Error installing CFclient

### DISCUSSION THREAD

**Discussion by Allanadn** [bitcraze/discussions#904] [Q&A]
Dear all,

I am new to crazyflies and am following the steps provided here: https://www.bitcraze.io/documentation/tutorials/getting-started-with-crazyflie-2-x/ to install the client in Windows. Upon running the command: pip3 install cfclient, I get the following error: 

![image](https://github.com/bitcraze/discussions/assets/33231979/21103463-5aa1-44b7-8ea6-6736308552aa)

Any ideas on how I could solve it?

**Comment by krichardsson** [comment]
Not sure what is happening, which python version are you using?

```
python --version
```
or 
```
python3 --version
```

**Reply by Allanadn** [reply]
Thank you for your answer @krichardsson! I am using  Python 3.10.11. After installing PyYaml 5.4.1, it sarted working. Nonetheless, my problem now is that I can cold boot the crazyflie but cannot connect to it...

**Reply by matejkarasek** [reply]
On Windows 11, I had to install the Visual Studio Build Tools 2022 with the following settings, that did the trick for me, it was a similar error I believe.
![visualstudiobuildtools](https://github.com/bitcraze/discussions/assets/10943061/93284152-cbed-428a-98b6-4f75eade186d)

**Reply by Allanadn** [reply]
Thanks for the help mate! Also installed it now. The scan problem still persists though :/

**Reply by krichardsson** [reply]
Great that the client problem is solved!
Please start a new discussion for the connection problem [as this is a different problem](https://github.com/orgs/bitcraze/discussions/55)

**Comment by matejkarasek** [comment]
Oh, I didn't read your previous comment. The Visual Studio Build tools fixed it for me with regural (recent) Python, I have not installed PyYaml.

**Reply by Allanadn** [reply]
No worries @matejkarasek , thank you for the tip anyways :) !
